### ISSUE TITLE
Problems with reproducing experimental results on the VOC dataset

### ISSUE THREAD

**Issue by tara0907** [Wu0409/DuPL#13]
Dear author, thank you for open-sourcing this work. While reproducing your experiment, I'd like to first replicate the results without using the ImageNet-21k pretrained parameters. When I set --pretrained=False in the train_final_voc.py file and ran training, my results for the first 2000 iterations were all in the single digits. I checked your training logs in the log folder and found that the value of pretrained is True in both dupl_train_voc/train.log and dupl_train_voc_21k/train.log, which is puzzling. How can I modify this to train the model without pretrained weights?

**Comment by Wu0409** [comment]
Hi, thanks for your interest in our work!

Our default WSSS setup requires starting with pre-trained weights (e.g., ImageNet-1k or ImageNet-21k). All experiments are configured this way, and training will likely fail without them.

If you'd like to train from scratch, you'll need to first pre-train a basic classification model and then load those weights before starting the DuPL training pipeline.

**Comment by tara0907** [comment]
Thank you very much for your reply! I understand that your models require pre-trained weights. In Table 2, you indicate that DuPL+ uses pre-trained weights for ImageNet-21k. So, are the DuPL results obtained using pre-trained weights for ImageNet-1K?

I didn't make any code changes and used the default command provided in your README file: python -m torch.distributed.run --nproc_per_node=2 train_final_voc.py --data_folder ./VOCdevkit/VOC2012/ . After training the model, I got the following results:

<img width="751" height="165" alt="Image" src="https://github.com/user-attachments/assets/a7f274f3-12f6-496e-b93c-f9d810b77605" />

I'm not familiar with the Deit model, and I can't tell whether the results I'm reproducing are from DuPL (which differs from your original accuracy by about 0.5 points) or from DuPL+ training (which differs from your original accuracy by about 1 point).

**Comment by Wu0409** [comment]
1. Regarding the DuPL results, we have two sets: one using the ImageNet-1K pre-trained weights and another using the ImageNet-21K weights. You can see both in the final two rows of Table 2, and the ImageNet-21K result is specifically marked.

2. I'm not entirely clear on what you mean by "DuPL+ training." The repo includes the training logs for both the ImageNet-1K and ImageNet-21K models, which you can use to compare their results. Generally, a variance of about 0.5 between runs is empirically acceptable.

**Comment by tara0907** [comment]
Thank you very much for your reply, I have got the answer I wanted, thank you again!

**Comment by Wu0409** [comment]
Feel free to reach out if you have any other questions~
